frontend/public/components/machine-config-pool.tsx

```diff
@@ -236,6 +260,29 @@ const pages = [
   navFactory.events(ResourceEventStream),
 ];
 
+const MachineConfigPoolUpdateStatus: React.FC<MachineConfigPoolUpdateStatusProps> = ({ obj }) => {
+  const { t } = useTranslation();
+  switch (getMachineConfigPoolUpdateStatus(obj)) {
+    case MCPUpdateStatus.Paused:
+      return (
+        <>
+          <PauseCircleIcon /> {t('public~Paused')}
+        </>
+      );
+    case MCPUpdateStatus.Updating:
+      return (
+        <>
+          <SyncAltIcon className="fa-spin co-icon-space-r" />
+          {t('public~Updating')}
+        </>
+      );
+    case MCPUpdateStatus.Updated:
+      return <UpToDateMessage />;
+    default:
+      return null;
```

Comment:
`<LoadingInline />` instead to avoid the empty state?

Reply:
https://user-images.githubusercontent.com/895728/168802747-b9dcb992-f0e2-4925-8548-ea32003ccdb1.mov

Reply:
So if the value is `null` because you didn't hit any of the if statements in `getMachineConfigPoolUpdateStatus`, what does that mean in relation to the MachineConfigPool? Is this a state we don't care about? Is it just not in a valid state?

I ask because it seems odd that a status getting method (`getMachineConfigPoolUpdateStatus`) doesn't return a state that causes this issue.

Reply:
> Is this a state we don't care about?

Exactly.  We're being a bit reductive with the conditions in order to consolidate the conditions in to `Update status` as a single field.  In reality the conditions are a matrix, but we're reducing them to a single state for the purposes of making `Update status`.